From f4b507116d988b7defcd7fac9a4be499d6ada9b7 Mon Sep 17 00:00:00 2001
From: SeongJae Park <sj@kernel.org>
Date: Thu, 11 Apr 2024 16:04:44 -0700
Subject: [PATCH] ==== commit cleanup ====

mm/damon: cleanup DAMON parameters commit logic

DAMON context is being used for receiving user's request for
monitoring/DAMOS, maintaining internal operation status, and returning
monitoring resutls/DAMOS statistics.  It was simple enough at the
beginning and therefore DAMON kernel API users were recommended to
manually modify the struct.  As DAMON evolves, the struct became much
more complicated and number of DAMON kernel API users also increased.
The increased complexity indeed caused some bugs especially related to
the DAMON parameters online update.

Add a standardized way for the DAMON/DAMOS user input parameters
updating.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 damon_meta_changes/RYnTeJnM | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 damon_meta_changes/RYnTeJnM

diff --git a/damon_meta_changes/RYnTeJnM b/damon_meta_changes/RYnTeJnM
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.39.2

